pkg/apis/externaldns/types_test.go

```diff
@@ -75,7 +75,7 @@ var (
 		AzureSubscriptionID:         "",
 		CloudflareProxied:           false,
 		CloudflareDNSRecordsPerPage: 100,
-		CloudflareRegionKey:		 "",
+		CloudflareRegionKey:         "",
```

Comment:
Please avoid unrelated changes. Consider filing a separate PR for linting instead